switchbot/devices/blind_tilt.py

```diff
@@ -0,0 +1,59 @@
+"""Library to handle connection with Switchbot."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from .curtain import SwitchbotCurtain, CURTAIN_EXT_SUM_KEY
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SwitchbotBlindTilt(SwitchbotCurtain):
+    """Representation of a Switchbot Blind Tilt."""
```

Comment:
Here's a proposal implementation to add open/close_up/close_down methods. I've added a close method that will finish closing the blinds in the direction they are already partially open, defaulting to close down when fully open (50%).

```suggestion
BLIND_COMMAND = "4501"
OPEN_KEYS = [
    f"{REQ_HEADER}{BLIND_COMMAND}010132",
    f"{REQ_HEADER}{BLIND_COMMAND}05ff32",
]
CLOSE_DOWN_KEYS = [
    f"{REQ_HEADER}{BLIND_COMMAND}010100",
    f"{REQ_HEADER}{BLIND_COMMAND}05ff00",
]
CLOSE_UP_KEYS = [
    f"{REQ_HEADER}{BLIND_COMMAND}010164",
    f"{REQ_HEADER}{BLIND_COMMAND}05ff64",
]

class SwitchbotBlindTilt(SwitchbotCurtain):
    """Representation of a Switchbot Blind Tilt."""

    @update_after_operation
    async def open(self) -> bool:
        """Send open command."""
        return await self._send_multiple_commands(OPEN_KEYS)

    @update_after_operation
    async def close_up(self) -> bool:
        """Send close up command."""
        return await self._send_multiple_commands(CLOSE_UP_KEYS)

    @update_after_operation
    async def close_down(self) -> bool:
        """Send close down command."""
        return await self._send_multiple_commands(CLOSE_DOWN_KEYS)

    @update_after_operation
    async def close(self) -> bool:
        """Send close command."""
        if (self.get_position() > 50):
            return await self.close_up()
        else:
            return await self.close_down()
```

Also needs the following imports:
```
from switchbot.devices.device import REQ_HEADER, update_after_operation
```